source/guides/ember-cli/installing.md

```diff
@@ -0,0 +1,62 @@
+## Got Node (and NPM)?
+
+If you're not sure if you have node.js installed, try running the following from your command line:
+
+```bash
+node --version
```

Comment:
Interesting... so on Windows with the latest nodejs, I only have 1.4.28 when I run ```npm --version```